courses/machine_learning/time_series_prediction/modeling_bqml.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>nit: add period to end of comments</p><p> </p><p>remove reference to MLE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>